config/static.c

```diff
@@ -38,8 +38,14 @@
 #define RTE_LOGTYPE_CONFIG RTE_LOGTYPE_USER2
 
 /* TODO Get the install-path via Makefile. */
-#define LUA_BASE_DIR "./lua/"
-#define LUA_CONFIG_FILE     "gatekeeper_config.lua"
+#define LUA_BASE_DIR               "./lua/"
+#define LUA_CONFIG_DIR             "./lua/config/"
+#define GATEKEEPER_CONFIG_FILE     "gatekeeper_config.lua"
+
+enum GATEKEEPER_COMPONENT_STATUS {
```

Comment:
We don't need this `enum`. As I describe in the previous commit, if the Lua script cannot initialize Gatekeeper, Gatekeeper will just shut down.